demos/device_shadow_for_aws/shadow_demo_main.c

```diff
@@ -760,10 +760,10 @@ int RunDeviceShadowDemo( bool awsIotMqttMode,
         /* This demo performs only Device Shadow operations. If matching the Shadow
          * MQTT topic fails or there are failure in parsing the received JSON document,
          * then this demo was not successful. */
-        if( ( lUpdateAcceptedReturn != EXIT_SUCCESS ) || ( lUpdateDeltaReturn != EXIT_SUCCESS ) )
+        if( ( lUpdateAcceptedReturn != pdPASS ) || ( lUpdateDeltaReturn != pdPASS ) )
         {
             LogError( ( "Callback function failed." ) );
-            returnStatus = EXIT_FAILURE;
+            returnStatus = pdFAIL;
         }
     }
 
```

Comment:
The return value of the demo can be provided as `return ( ( xDemoStatus == pdPASS )? EXIT_SUCCESS : EXIT_FAILURE );`
as the return value should be `EXIT_SUCCESS/EXIT_FAILURE`.